11/jdk/slim-aws/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM openjdk:11.0.1-jdk-slim
+MAINTAINER Forest Keepers
+
+# Default to UTF-8 file.encoding
+ENV LANG C.UTF-8
+
+ENV AWSCLI_VERSION 1.16.47
+
+RUN apt-get update && \
+    apt-get install -y \
+      bzip2 \
```

Comment:
Do we need zip in this image?

Reply:
I've removed `zip` and `unzip` and ran some tests! Now we don't need them.